### PR TITLE
Add workingDirectory option for publishing a subdirectory

### PR DESCRIPTION
**Pull request by fshot**
This addresses https://github.com/cloudflare/wrangler-action/issues/3

Usage is as suggested in the issue:

```
- name: Publish worker one
  uses: cloudflare/wrangler-action@1.0.0
  with:
    apiKey: ${{ secrets.CF_API_KEY }}
    email: ${{ secrets.CF_EMAIL }}
    workingDirectory: 'worker-one'
- name: Publish worker two
  uses: cloudflare/wrangler-action@1.0.0
  with:
    apiKey: ${{ secrets.CF_API_KEY }}
    email: ${{ secrets.CF_EMAIL }}
    workingDirectory: 'worker-two'
```

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -30,10 +30,19 @@ export CF_API_KEY="$INPUT_APIKEY"
...
 if [ -z "$INPUT_ENVIRONMENT" ]
 then
   wrangler publish
 else
   wrangler publish -e "$INPUT_ENVIRONMENT"
 fi
 
+if ! [ -z "$INPUT_WORKINGDIRECTORY" ]
+then
+  cd $HOME
```
Comment (signalnerve) [inline entrypoint.sh]: Ah, good catch here! I would have forgotten this :)

**Comment by fshot** [comment]
Hi @signalnerve , we are only using this in private repos right now, but I can share what the Github action workflow yaml looks like:

```  
name: Deploy subfolder to production

on:
  push:
    branches:
    - release
    paths:
    - 'subfolder-to-deploy/**'

jobs:
  deploy:
    runs-on: ubuntu-latest
    name: Deploy
    steps:
      - uses: actions/checkout@master
      - name: Publish
        id: publish
        uses: replicatedhq/wrangler-action@master
        with:
          apiKey: ${{ secrets.CF_API_KEY }}
          email: ${{ secrets.CF_EMAIL }}
          environment: 'production'
          workingDirectory: subfolder-to-deploy
```

**Comment by signalnerve** [comment]
Hey @fshot, sorry for the delay on reviewing this! I'm testing this out this morning, I'll let you know how it goes.

**Comment by fshot** [comment]
Thanks for the update -- no rush, we would love to see it merged in, but are doing just fine using a fork in the meantime.

Cheers,
Frank

**Comment by fshot** [comment]
My pleasure, and thanks for making the integration.  Cheers!

**Pull request merged**